docs/articles/getting-started/recovery.md

```diff
@@ -0,0 +1,66 @@
+One of the powers of Bytewax is the ability to develop dataflows that
+accumulate state as they process data. By default, this state is just
+stored in memory and will be lost if the dataflow is restarted or
+there is a fault. Bytewax provides some tools and techniques to allow
+you to **recover** your state and resume the dataflow after a fault.
+
+## Prerequisites
+
+Not all dataflows are possible to be recovered. There are a few
+requirements:
+
+1. _Stateful Operators_ - Only the operators
+   [reduce](./operators#reduce) and [stateful
+   map](./operators#stateful_map) aggregate state _between_ epochs. If
+   your dataflow is simple enough to not be using these,
```

Comment:
I guess this point actually isn't true. You can recover a dataflow that isn't using any recoverable operators, just the execution won't result in any data being in the recovery store and so recovery is "trivial" in that you can just replay input items without setting up state again. Since Bytewax does all the right things for you, I think I'll just drop this point.